app/src/main/java/org/mozilla/tv/firefox/pinnedtile/PinnedTileAdapter.kt

```diff
@@ -52,6 +53,7 @@ class PinnedTileAdapter(
 
     private val uiScope = CoroutineScope(Dispatchers.Main + uiLifecycleCancelJob)
 
+    @ExperimentalCoroutinesApi
```

Comment:
Do we need this for `onBindViewHolder`?

Reply:
It's needed for line 65 `onBindCustomHomeTile(uiScope, holder, item)`